tsdb/index/tsi1/partition.go

```diff
@@ -111,6 +110,17 @@ func NewPartition(sfile *tsdb.SeriesFile, path string) *Partition {
 	return p
 }
 
+// SetMaxLogFileSize provides a setter for the partition setting of maxLogFileSize
+// that is otherwise only available at creation time. Returns the previous value.
+// Only for tests!
+func (p *Partition) SetMaxLogFileSize(new int64) (old int64) {
+	p.mu.Lock()
+	old = p.maxLogFileSize
+	p.maxLogFileSize = new
```

Comment:
You can do this more idiomatically with 

`old, p.maxLogFileSize = p.maxLogFileSize, new`

Reply:
👍 Sounds good